stablehlo/reference/Ops.cpp

```diff
@@ -224,6 +224,17 @@ SmallVector<Tensor> eval(
       Tensor runtimeOperand = scope.find(realOp.getOperand());
       Tensor runtimeResult = evalRealOp(runtimeOperand, realOp.getType());
       scope.add(op.getResults(), {runtimeResult});
+    } else if (auto reduceOp = dyn_cast<ReduceOp>(op)) {
+      SmallVector<Tensor> runtimeInputs = scope.find(reduceOp.getInputs());
```

Comment:
Let's drop the "runtime" prefixes from names and use `auto` where applicable, to be consistent with the recent refactoring / cleanup.